tests/assets/sign_ansible.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+FILE_PATH=$1
+SIGNATURE_PATH="$1.asc"
+
+GPG_KEY_ID="Pulp QE"
```

Comment:
Don't we also get the key id passed down from the signing service?
I'm wondering if we could write a generic script for at least simple gpg signing.

Reply:
Yes, we can. (I copied this from pulp_ansible) I'll change it to be generic to allow for more signing-services to be created.

Reply:
Well actually i was thinking that we should ship the very basic script with pulpcore at some point.